keras/utils/generic_utils.py

```diff
@@ -208,7 +208,7 @@ class Progbar(object):
     """Displays a progress bar.
 
     # Arguments
-        target: Total number of steps expected.
+        target: Total number of steps expected, -1 if unknown.
```

Comment:
The reason I used -1 is due to the reporthook parameter in `urlretrieve`, which in this case is the function `dl_progress`:

> If reporthook is given, it must be a function accepting three numeric parameters: A chunk number, the maximum size chunks are read in and the total size of the download (-1 if unknown).

A added a 289a5e4 so both `None` and -1 are accepted, which also means less code has to change. That work?